helm/layers-api/templates/ingress.yaml

```diff
@@ -14,7 +14,10 @@ metadata:
     nginx.ingress.kubernetes.io/server-snippet: |
 {{- else }}
     nginx.org/client-max-body-size: 5m
+    nginx.org/location-snippets: |
+      include CORS_02_location.inc.conf;
     nginx.org/server-snippets: |
+      set $cors_config_file '/etc/nginx/default.cors.json';
```

Comment:
Are these files present in publicly available nginx ingress controller?
if not - this seems to break this ingress for anyone outside Kontur. I'd advice to parameterise this via a separate helm property.

FYI @Komzpa

Reply:
https://konturio.slack.com/archives/C014AF8MFRD/p1671101846650499

Reply:
I have added a compatibility layer with Flux